Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs

```diff
@@ -12,3 +13,47 @@ let ( / ) a b =
     ((parsed a) / (parsed b)).InnerSimplified
 let ( ** ) a b =
     ((parsed a).Pow(parsed b)).InnerSimplified
+let ( ~~~ ) a = 
+    (negation (parsed a)).InnerSimplified
+let ( ||| ) a b = 
+    (union (parsed a) (parsed b)).InnerSimplified
+let ( ^^^ ) a b = 
+    (intersect (parsed a) (parsed b)).InnerSimplified
+let ( = ) a b = 
```

Comment:
Doesn't it override assignment operator? If it doesn't, then it's probably ok

Reply:
The assignment operator in F# is `<-`.